src/team-extension.ts

```diff
@@ -176,7 +183,9 @@ export class TeamExtension  {
     //Opens the file history page for the currently active file
     public OpenFileHistory(): void {
         if (this.ensureInitialized()) {
-            this._gitClient.OpenFileHistory(this._gitContext);
+            if (this._gitClient) {
+                this._gitClient.OpenFileHistory(this._repoContext);
+            }
```

Comment:
I took a note for this.  It may make sense to indicate the type of repo a particular command applies to.